packages/components/Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV NPM_CONFIG_LOGLEVEL warn
 COPY package.json package.json
 COPY packages/components/package.json packages/components/package.json
 COPY packages/gateway/package.json packages/gateway/package.json
-COPY packages/commons/package.json packages/commons/package.json
+COPY packages/commons packages/commons
```

Comment:
Can no longer build components without all the tsconfig inputs and typescript files.  No idea why but I have to add the whole package